docs/Documentation/viz_analytics/index.md

```diff
@@ -0,0 +1,35 @@
+#Visualization and Analytics Software Tools for Eagle System
```

Comment:
If we are including these tools on the Visualization intro page, shouldn't they be in the Visualization section? 
I also think that we should remove references to Eagle in the intro to this page, since we would probably want to include visualization tools here that are on other systems.